modules/standard/Math.chpl

```diff
@@ -149,11 +149,31 @@ module Math {
     return fabsf(_i2r(im));
   }
 
-  /* Returns the real magnitude of the complex argument `x`.
+  /* Returns the real magnitude of the complex argument `z`.
 
      :rtype: The type of the real component of the argument (== `w`/2).
   */
-  inline proc abs(x : complex(?w)) return sqrt(x.re*x.re + x.im*x.im);
+  inline proc abs(z : complex(?w)) {
+    extern proc cabsf(z: complex(64)): real(32);
+    extern proc cabs(z: complex(128)): real(64);
+    if w == 64 then
+      return cabsf(z);
+    else
+      return cabs(z);
+  }
+
+
+  /* Returns the real phase angle of complex argument `z`. */
+  inline proc arg(z: complex(64)): real(32) {
+    extern proc cargf(z: complex(64)): real(32);
+    return cargf(z);
+  }
+
+  /* Returns the real phase angle of complex argument `z`. */
+  inline proc arg(z: complex(128)): real(64) {
+    extern proc carg(z: complex(128)): real(64);
+    return carg(z);
+  }
```

Comment:
Why not combine the two arg methods into one, like with abs? If not' I'd consider no-docing one of them.